views/scan.hbs

```diff
@@ -15,7 +15,15 @@
       {{else}}
         <h2 class="headline scan-results-headline">{{{ fluentNestedBold "scan-results-known-breaches" breachCount=foundBreaches.length }}}</h2>
         <button id="new-scan-page"
-          class="open-oauth temp-marketing-btn-blue button-top" {{> analytics/fxa id="fx-monitor-alert-me-blue-btn-top" }} data-event-category="Alert me about new breaches"> {{ getString "alert-about-new-breaches" }} </button>
+          class="open-oauth temp-marketing-btn-blue button-top"
+          {{#if experimentFlags.controlBranch }}
+            {{> analytics/fxa id="fx-monitor-scan-alert-me-blue-btn-top-exp4v2" }}
+          {{else if experimentFlags.treatmentBranch }}
+            {{> analytics/fxa id="fx-monitor-scan-alert-me-blue-btn-top-unchecked-exp4v2" }}
```

Comment:
So the control branch is checked and the treatment branch is unchecked?

Reply:
Control branch in this case doesn't have a checkbox period, thus no "check" status.

Control: 

<img width="998" alt="image" src="https://user-images.githubusercontent.com/2692333/86015236-eb542b80-b9e6-11ea-822b-ae72b40f8c25.png">